handlers/simulation.js

```diff
@@ -0,0 +1,232 @@
+const DOWN_PAYMENT_FIXED_FIELD = "fixedValue";
+const DOWN_PAYMENT_TYPE = {
+    inAmount: "IN_AMOUNT",
+    overAmount: "OVER_AMOUNT",
+};
+
```

Comment:
Please add a TODO section to ensure these code blocks will be removed when the simulation API is ready in the FI adapter.